mobile/src/libs/components/icon/icon.tsx

```diff
@@ -7,12 +7,13 @@ import { iconNameToIcon } from './libs/maps/maps';
 type Properties = {
   name: IconName;
   color?: string;
+  styles?: React.CSSProperties;
```

Comment:
we should not pass styles to our components, if you need to add margins/paddings or positioning then wrap it in View, in case you need specific styles - pass specific properties
```suggestion
```